test/test_helpers/pact_helper.rb

```diff
@@ -1,7 +1,15 @@
+def publishing_api_port
```

Comment:
Would it be worth it to make this a constant so it's obvious that it's not dynamic?